### PR TITLE
[CLI] - Set package.json name to the name the user entered for the project

### PR DESCRIPTION
**Pull request by b4s36t4**
Solves the issue #84

### CODE REVIEW

File: packages/cli/src/create/helpers/create-app.ts [hunk 1]
```diff
@@ -141,6 +160,23 @@ export async function createApp({
...
+    const packageString = fs.readFileSync(jsonFile);
+    try {
+      const packageJson = JSON.parse(packageString.toString());
+      if (packageJson) {
+        packageJson.name = appName;
+      }
+
+      fs.writeFileSync(jsonFile, JSON.stringify(packageJson, null, 2) + os.EOL);
+
+      console.log("Updated package.json.");
```
Comment (jarrodwatts) [inline packages/cli/src/create/helpers/create-app.ts:173]: Can we remove these console logs? same with the catch block below.

File: packages/cli/src/create/helpers/create-app.ts [hunk 2]
```diff
@@ -141,6 +160,23 @@ export async function createApp({
...
+      console.log();
+    } catch (e) {
+      console.log("Reading JSON error");
```
Comment (jarrodwatts) [inline packages/cli/src/create/helpers/create-app.ts:176]: We should handle the error here.

I don't think failing to rename the package name is severe enough to exit the process, but we should maybe just console.log this - "Something went wrong renaming package, skipping..."

**Comment by jarrodwatts** [comment]
Thanks Mahesh! Looks good!

A few comments from me and I've asked our engineer @joaquim-verges to take a look too

**Comment by jnsdls** [comment]
Thanks for the PR! Please add a changeset (using `yarn changeset`) that briefly describes what these changes mean to a user.

In this case it can probably be a `patch` version change.

**Pull request merged**